detect/utils.go

```diff
@@ -122,7 +122,14 @@ func printFinding(f report.Finding) {
 		Italic(true).
 		Foreground(lipgloss.Color("#f05c07"))
 	matchEnd := lipgloss.NewStyle().SetString(f.Match[secretInMatchIdx+len(f.Secret):]).Foreground(lipgloss.Color("#f5d445"))
+
+	lineEndIdx := matchInLineIDX + len(f.Match)
+	if len(f.Line)-1 <= lineEndIdx {
+		lineEndIdx = len(f.Line) - 1
+	}
+
 	lineEnd := f.Line[matchInLineIDX+len(f.Match):]
+
```

Comment:
@zricethezav 
As noted in #1145, this PR didn't solve the issue.
Maybe you meant to fix the issue as suggested below?
```suggestion

	lineEndIdx := matchInLineIDX + len(f.Match)
	if len(f.Line)-1 <= lineEndIdx {
		lineEndIdx = len(f.Line) - 1
	}

	lineEnd := f.Line[lineEndIdx:]
```